pom.xml

```diff
@@ -71,13 +64,13 @@
         <dependency>
             <groupId>org.zalando.stups</groupId>
             <artifactId>stups-spring-oauth2-client</artifactId>
-            <version>1.0.11</version>
+            <version>1.0.14</version>
         </dependency>
 
         <dependency>
             <groupId>org.zalando.stups</groupId>
             <artifactId>stups-spring-oauth2-server</artifactId>
```

Comment:
Probably this is not needed at all, because we are not exposing an oauth secured api